### PR TITLE
Change font-weight of headings and texts

### PR DESCRIPTION
**Pull request by Sanji515**
#### Changes proposed in this pull request:

- Changed font-weight of headings to 400 and normal texts to 300.

- Link to live demo: http://pr-198-evalai.surge.sh

### CODE REVIEW

File: src/app/components/about/about.component.scss [hunk 1]
```diff
@@ -19,14 +19,14 @@
   color: #4d4d4d;
   margin: 1.14rem 0 0.912rem 0;
   font-size: 34px;
-  font-weight: 500;
+  font-weight: 400;
```
Comment (lunayach) [inline src/app/components/about/about.component.scss]: I see **400** font-weight used at other places as well. Can we use some variable like _$bolder_ / _$boldest_ etc. (depending on the other values of weight being used)

Also, at other places.

Comment (Sanji515) [reply src/app/components/about/about.component.scss]: yes we already have that one, will do that. Thanks :)

Comment (Sanji515) [reply src/app/components/about/about.component.scss]: @lunayach done

**Comment by Sanji515** [comment]
@RishabhJain2018 @lunayach Please review

**Review by lunayach — APPROVED** [review]

**Pull request merged**